scriptmodules/emulators/dosbox-staging.sh

```diff
@@ -13,10 +13,14 @@ rp_module_id="dosbox-staging"
 rp_module_desc="modern DOS/x86 emulator focusing on ease of use"
 rp_module_help="ROM Extensions: .bat .com .exe .sh .conf\n\nCopy your DOS games to $romdir/pc"
 rp_module_licence="GPL2 https://raw.githubusercontent.com/dosbox-staging/dosbox-staging/master/COPYING"
-rp_module_repo="git https://github.com/dosbox-staging/dosbox-staging.git master"
+rp_module_repo="git https://github.com/dosbox-staging/dosbox-staging.git :_get_branch_dosboxstaging"
 rp_module_section="exp"
 rp_module_flags="sdl2"
 
+function _get_branch_dosboxstaging() {
```

Comment:
Although this works this would be better called

`_get_branch_dosbox-staging`

Reply:
Absolutely right. All fixed now. Thanks!